orcid-api-common/src/main/java/org/orcid/api/common/jaxb/OrcidJacksonJaxbJsonProvider.java

```diff
@@ -31,8 +31,8 @@
  * @author Will Simpson
  */
 @Provider
-@Consumes({ VND_ORCID_JSON, ORCID_JSON, "text/orcid+json" })
-@Produces({ VND_ORCID_JSON, ORCID_JSON, "text/orcid+json" })
+@Consumes({ MediaType.APPLICATION_JSON })
+@Produces({ MediaType.APPLICATION_JSON })
```

Comment:
@rcpeters See also OrcidJacksonJaxbJsonProviderPretty. The other media types are pretty, whereas application/json is not. It looks like application/json was working by 'luck' previously, and that's why it changed when i removed the default Jersey providers.